docs/source/tensorbay_cli/tbrn.rst

```diff
@@ -1,3 +1,96 @@
 #########################
  TensorBay Resource Name
 #########################
+
+TensorBay Resource Name(TBRN) uniquely defines the resource stored in TensorBay.
+TBRN begins with ``tb:``. It can represent different levels of resources in TensorBay:
+
+| 1. Dataset level
+
+.. code:: console
+
+    tb:[dataset_name]
+
+For example, the following TBRN means the dataset "VOC2012".
+
+.. code:: console
+
+    tb:VOC2012
+
+
+| 2. Segment level
+
+.. code:: console
+
+    tb:[dataset_name]:[segment_name]
+
+For example, the following TBRN means the "train" segment of dataset "VOC2012".
+
+.. code:: console
+
+    tb:VOC2010:train
+
+
+| 3. File level
+
+.. code:: console
+
+    tb:[dataset_name]:[segment_name]://[remote_path]
+
+For example, the following TBRN means the file "2012_004330.jpg" of the "train" segment in normal dataset "VOC2012".
+
+.. code:: console
+
+    tb:VOC2012:train://2012_004330.jpg
+
+Moreover, the file folder is supported by adding "/" at the end of remote_path.
```

Comment:
Moreover, there can be hierarchy information in remote_path. Different levels are separated by "/".

.. code:: console

   tb:[dataset_name]:[segment_name]://[level_0/level_1/.../file_name]

For example, the following TBRN is legal.

.. code:: console

   tb:my_dataset:train://train1/000001.jpg

We can obtain all data whose remote paths are prefixed by ``train1`` with the following TBRN:

.. code:: console

   tb:my_dataset:train:train1/